integrationtest/docker/jdk8/start.sh

```diff
@@ -36,12 +36,19 @@ case "$COMMAND" in
     JAVA_OPTS="$JAVA_OPTS -Dgearpump.hostname=$(hostname)"
     sh "$SUT_HOME"/bin/"$COMMAND" "$@"
     ;;
+  storm-drpc)
+    STORM_CONFIG="$SUT_HOME"/storm.yaml
+    echo "drpc.servers:" > "$STORM_CONFIG"
+    echo "      "-" "\"`ip route | awk '/default/ {print $3}'`\" >> "$STORM_CONFIG"
```

Comment:
ip address of docker0 is the docker host on linux, which is how the host connect ports exposed by containers inside docker0